uci_ssh_test.go

```diff
@@ -0,0 +1,67 @@
+package uci_test
+
+import (
+	"testing"
+
+	"github.com/digineo/go-uci"
+	"github.com/stretchr/testify/assert"
+)
+
+const USERNAME = "root"
+const PASSWORD = ""
+const HOST = "192.168.1.129"
+
+func TestWrapperConnect(t *testing.T) {
+	// ToDo: we should ship a test server!
+	dut := uci.NewSshTree(USERNAME, PASSWORD, HOST)
+	err := dut.Disonnect()
+	if err != nil {
+		t.Error(err.Error())
+	}
+}
+
+func TestInterface(t *testing.T) {
+	assert := assert.New(t)
+	v := &uci.SshTree{}
+	var i interface{} = v
+	_, ok := i.(uci.Tree)
+	assert.Equal(ok, true, "Interface not implemented")
+}
```

Comment:
This can be condensed to

```go
var _ uci.Tree = (nil)(*uci.SshTree)
```

and moved below the `SshTree` definition.

You won't even able to compile, if that interface isn't implemented.

Reply:
Indeed. But I like this kind of implementation. This makes it a bit more easy to work on multiple things. If the compile will not work it is always annoying.

Reply:
But the implementation is already there.